docs/blog/2025-community-update.rst

```diff
@@ -0,0 +1,91 @@
+:og:image: https://www.writethedocs.org/_static/logo-opengraph.png
+
+.. post:: Aug 8, 2025
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - August 2025
+=============================================
+
+Hey there documentarians!
+
+Eric Holscher here, sharing this year's community update as we reflect on another year of growth and change for Write the Docs.
+
+We've been heavily focused on conferences this year, but our Slack community, Newsletter, Meetups, and other initiatives continue without too much change. 
+
+Portland was a great success
+----------------------------
```

Comment:
```suggestion
WTD Portland was a great success
-------------------------------
```

Reply:
This feels repetitive? I think people realize we're talking about ourselves?

Reply:
For sure, but it just reads odd as if it's the city. Could use something like 

> The Portland conference was a great success 

instead?